backend/app/routes/subscribers/postSubscriber.js

```diff
@@ -2,8 +2,22 @@ const to = require("await-to-js").default;
 const Subscriber = require('../../models/Subscriber');
 const { ErrorHandler } = require('../../../helpers/error')
 const constants = require('../../../constants');
+const nodemailer = require('nodemailer')
+const config = require('../../../config')
+const { welcomeSubscriberMailTemplate } = require('../../../utility/emailTemplates')
 
 module.exports = async (req, res, next) => {
+  const transporter = nodemailer.createTransport({
+    type: 'SMTP',
+    host: config.EMAIL_HOST,
```

Comment:
can we add env.sample file and add these env variables with required values/placeholders(in case of credential)

Reply:
I think no need for `env.sample` file because, `.env` file already there with required values/placeholders.

Reply:
Then add the new env variables that got introduced in this pr there itself. It will be easier for new developers

Reply:
There is no new varaible used here and they already exists in `.env` file